book.rb

```diff
@@ -0,0 +1,28 @@
+require './rental'
+require 'json'
+
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rentals(date, person, _book)
+    Rental.new(date, person, self)
+  end
```

Comment:
- [ ] There is no need to the parameter `_book` since it's not used in the method.